applications/GeoMechanicsApplication/custom_elements/stress_state_policy.cpp

```diff
@@ -24,7 +24,16 @@ Vector StressStatePolicy::DefineVoigtVector(std::size_t Dimension)
     return VoigtVector;
 }
 
+Vector StressStatePolicy::DefineInterfaceVoigtVector(std::size_t Dimension)
+{
+    Vector VoigtVector = ZeroVector(GetInterfaceVoigtSize(Dimension));
+    VoigtVector[0]     = 1.0;
+
+    return VoigtVector;
+}
```

Comment:
This is certainly correct for 2D, but not sure yet if it's also correct for 3D

Reply:
Yes, this is correct for 3D  planar and point  interfaces. They have 1 normal and 2 shear directions.
The question is very valid though. A 3D line interface could be either 1 normal and 2 shear directions ( e.g. a mortar joint in brickwork ) or 2 normal and 1 shear directions ( e.g. a pile in soil )

Reply:
Moved to the interface stress state